build/build.yml

```
@@ -23,8 +23,6 @@ steps:
   displayName: Restore nuget packages
 
 - pwsh: |
-    Install-PackageProvider -Name NuGet -Force
-    # install platyPS to gernate help docs from markup files
     Install-Module -Name platyPS -Repository PSGallery -SkipPublisherCheck -Force
```

If that doesn't work, you also might get rid of the explicit repo:
```suggestion
    Install-Module -Name platyPS -SkipPublisherCheck -Force
```
The default repo ordered list on a clean agent should be secure.